### ISSUE TITLE
Theme-builder for smaller screens

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#3232]
Right now we are breaking WCAG guidelines with our... special handling of smaller screen sizes in our theme builder. 

I got a feeling most people that are creating and building colorscales for code and Figma are using a bigger screen size. But if you are browsing our page with a phone it is a little weird that it totally fails when entering the theme builder. 

![Image](https://github.com/user-attachments/assets/d908c144-8246-42e9-bfe9-fc313f4311c4)

My suggestion is to make a alternative view for smaller screens that only show some important elements for browsing.

**Comment by mrosvik** [comment]
Suggestion from @Barsnes ; Use `dialog` on mobile to open the content from sidebar

**Comment by mrosvik** [comment]
@Febakke and @Thunear will create a suggestion

**Comment by Barsnes** [comment]
In PR [#3294](https://github.com/digdir/designsystemet/pull/3294), the themebuilder was made usable for smaller screens. No work was done to check if the userflow is any good, it was just to fix big issues.

**Comment by mimarz** [comment]
We have fixed the most critical issues, but in reality this tool is not intended for use on smaller screens due to its nature and output. 

Will look at potential other improvements in separate issues
